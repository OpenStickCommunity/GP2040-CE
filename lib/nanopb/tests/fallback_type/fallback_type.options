Message1.member2 fallback_type:FT_POINTER
