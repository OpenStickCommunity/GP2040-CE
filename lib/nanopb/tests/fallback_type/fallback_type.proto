syntax = "proto2";

message Message1
{
  required int32 member1 = 1; // must remain as int
  required string member2 = 2; // must become a pointer
  required string member3 = 3; // must become pb_callback_t
}

