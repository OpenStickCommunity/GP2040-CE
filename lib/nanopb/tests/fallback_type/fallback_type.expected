int32_t member1;
char \*member2
pb_callback_t member3
